shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/check/consistency/CRC32MatchMySQLSingleTableDataCalculator.java

```diff
@@ -58,17 +58,17 @@ public String getAlgorithmType() {
     public Iterable<Object> calculate(final DataCalculateParameter dataCalculateParameter) {
         String logicTableName = dataCalculateParameter.getLogicTableName();
         List<Long> result = dataCalculateParameter.getColumnNames().stream().map(each -> {
-            String sql = SQL_BUILDER.buildSumCrc32SQL(logicTableName, each);
+            String sql = SQL_BUILDER.buildChecksumSQL(logicTableName, each);
```

Comment:
Why change method name to `check sum`, but class name still `CRC32`?

Reply:
Recovered to `buildCrc32SQL`.